tests/unittests/operators/types_tests.py

```
@@ -0,0 +1,46 @@
+
+class TableViewTests(unittest.TestCase):
+    def test_table_view_basic(self):
+        table = TableView()
+        table.add_column("column1", label="Column 1")
+        table.add_column("column2", label="Column 2")
+        assert table.keys() == ["column1", "column2"]
+
+        with self.assertRaises(ValueError):
+            table.add_column("column1", label="Column 3")
+
+        mock_on_click = lambda: None
```

Ruff

Do not assign a `lambda` expression, use a `def`

Rewrite `mock_on_click` as a `def`

(E731)